action.yml

```
@@ -11,6 +11,10 @@ inputs:
   repo-name:
     description: 'The name of the repository'
     required: true
+  repo-token:
+    description: 'The PAT used to authenticate to the repository.'
```

Refactor description

```suggestion
    description: 'The GitHub or PAT token used to authenticate to the repository.'
```